Goobi/config/log4j.properties

```diff
@@ -31,7 +31,7 @@ log4j.appender.stdout.layout.ConversionPattern=%-5p %d %C.%M(%F:%L)%n        %m%
 
 # ein Rolling-File-Appender (10 Backups)
 log4j.appender.rolling=org.apache.log4j.RollingFileAppender
-log4j.appender.rolling.File=/usr/local/logs/goobi.log
+log4j.appender.rolling.File=/usr/local/kitodo/logs/goobi.log
```

Comment:
What about switching from `goobi.log` to `kitodo.log`? That also requires an update in `build.xml`.